common/src/main/scala/explore/model/CallForProposal.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.model
+
+import cats.Eq
+import cats.derived.*
+import cats.syntax.all.*
+import io.circe.Decoder
+import lucuma.core.model.Semester
+import lucuma.schemas.decoders.given
+
+case class CallForProposal(semester: Semester) derives Eq, Decoder
```

Comment:
This will grow to include the call type, etc. right?

Reply:
Yes, I'm pretty sure  it will grow, we probably need partners and instruments to filter things on the UI